faster_raster_test.go

```diff
@@ -228,6 +242,18 @@ func Test_Processing(t *testing.T) {
 			raster.Stop()
 		})
 
+		Convey("returns an error when the SVG raster request is cancelled", func() {
+			err := raster.Run()
+			defer raster.Stop()
+			So(err, ShouldBeNil)
+
+			ctx, doneFunc := context.WithTimeout(context.Background(), 1*time.Millisecond)
+			defer doneFunc()
```

Comment:
It's not clear to me that it's necessary to store and call this?

Reply:
It's not. Doesn't do anything useful in this case. I'm removing it.